src/ui/components/statusrow/StatusRow.js

```diff
@@ -56,6 +57,13 @@ const STATUSES = [
         getDescription: () => "Radioactive",
         getTooltip: () => "Radioactive",
     },
+    {
+        key: STATUS_KEYS.HAZARDOUS,
+        shouldRender: (entity, entityType, hasHazards) => isCernMode && entityType === "equipment" && hasHazards,
```

Comment:
The isCernMode flag is used for CERN-specific functionalities that are not intended for open-source users. Since hazards is a default EAM functionality, we might want to enable it for open-source users.